kolibri/core/templates/kolibri/base.html

```diff
@@ -1,10 +1,8 @@
-{% load i18n kolibri_tags %}
-<html>
+{% load i18n kolibri_tags webpack_tags %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
-  {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
-  {% base_frontend_async %}
-  {% base_frontend_sync %}
+  {% webpack_assets 'default_frontend' %}
+  {% base_frontend_assets %}
```

Comment:
I've been struggling a bit with the names and purposes of `template_tags`, too. (discussion here: https://github.com/learningequality/kolibri/pull/78#discussion-diff-62903599)